fedimintd/src/distributedgen.rs

```diff
@@ -101,6 +102,12 @@ pub async fn run_dkg(
     )
     .await?;
 
+    // When testing on localhost, it's possible to get
```

Comment:
I'm so sloppy. I wrote all these flushing, to avoid this hack and yet I still pushed it.